site/docs/v1/tech/installation-guide/cloud.adoc

```diff
@@ -256,6 +257,65 @@ image::installation-guides/cloud/deployments-destroying.png[The deployment is be
+* Export your FusionAuth database to a file. When you run the `pg_dump` command, please use the following options: `--format=c` and `--compress 9` to minimize the file size.
+* Encrypt the export file with a symmetric solution such as gpg.
+* Purchase a FusionAuth Cloud deployment that meets your needs. Select the version of FusionAuth that is the same as your local installation. You will be able to upgrade FusionAuth later; see <<Upgrading a Deployment>> for more.
+* link:https://account.fusionauth.io/account/support/[Open a support ticket] and provide the database export file, the version of PostgreSQL you are running, and the name of the new FusionAuth deployment.
+* Provide the password for the database export through another channel. You can mention the channel you'd prefer in the support ticket. Slack, email and a separate support ticket are all options.
+
+Within 1-2 business days, we will upload your data to your FusionAuth Cloud deployment.
+
+After your data is uploaded, upgrade your deployment to any subsequent FusionAuth, if desired. See <<Upgrading a Deployment>> for more.
+
+.Migration checklist
+[cols="1,10"]
+|===
+| Done? 
+| Task
+
+| +++<i class="far fa-square"></i>+++
+| Confirmed running Postgres for local instance.
+
+| +++<i class="far fa-square"></i>+++
+| Found FusionAuth version for local instance.
+
+| +++<i class="far fa-square"></i>+++
+| Created a new deployment. Instructions: <<Creating a Deployment>>. 
+
+| +++<i class="far fa-square"></i>+++
+| Ensured the FusionAuth Cloud deployment and the local instance are running the same version of FusionAuth.
+
+| +++<i class="far fa-square"></i>+++
+| Communicated the version of Postgres used in the export file.
+
+| +++<i class="far fa-square"></i>+++
+| Encrypted the database export.
+
+| +++<i class="far fa-square"></i>+++
+| Provided the decryption password via a separate communication channel.
+
+|===
+
+
+==== Migrating Away From FusionAuth Cloud
+
+To migrate from FusionAuth Cloud to a self-hosted instance, please request a database export, as documented in <<Accessing User Data>>. After you have the data, install it in a PostgreSQL database and configure your self-hosted instance to point to that database. 
+
+Ensure your self-hosted FusionAuth instance uses the same FusionAuth and database version is the same as that used by your Cloud deployment. You can view your FusionAuth version in the administrative user interface. To find your deployment's database version, the best way currently is to open a support ticket. There's an https://github.com/FusionAuth/fusionauth-issues/issues/1390[open issue] for displaying the database version in the administrative user interface.
```

Comment:
This still reads off to me ... maybe just remove ~~is the same~~

or something else

Ensure your self-hosted FusionAuth instance uses the same FusionAuth and database version as used in your Cloud deployment.

Reply:
Thanks, modified to make this more clear.